netci.groovy

```diff
@@ -559,6 +559,10 @@ combinedScenarios.each { scenario ->
                             switch (architecture) {
                                 case 'x64':
                                 case 'x86':
+
+                                    //This will shorten the output file path lengths
```

Comment:
Can you elaborate in this comment on why this is being done? That way someone looking at this in the future will be able to reason about whether it can be removed.

Also, minor nit: add a space after the '//'.